### PR TITLE
Initial commit for the Japanese translation 1.4

### PR DESCRIPTION
**Pull request by TatchNicolas**
Tried to translate based on the expressions used in Japanese version 1.3 for consistency, but I paraphrased when I felt there's an better way.

 I'd like to have opinions of other translators, including translators for other languages.

### CODE REVIEW

File: version/1/4/ja/code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+* 歓迎的で開放的な言葉を使うこと
+* 異なる視点や経験を尊重する態度をとること
+* 建設的な批判を率直に受け入れること
+* 何がコミュニティにとって最適であるかに焦点を当てること
+* 他のコミュニティメンバーに共感すること
+
+受け入れられない行動の例としては以下のものがあります:
+
+* 性的な言葉・画像の使用および
```
Comment (TatchNicolas) [inline version/1/4/ja/code_of_conduct.md]: Oh yes I missed a part of the list item...I'll fix it soon. I should've checked more carefully...

File: version/1/4/ja/code_of_conduct.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+執行方針についての詳細は個別に提示されることがあります。
+
+行動規範に従わない、または良心に基づいた行動規範の執行をしないメンテナ
+は、プロジェクトの他のリーダーに決められた通りに一時的または恒久的な処
+罰を受けることがあります。
+
+## 帰属
+
+この行動規範は、[http://contributor-covenant.org/version/1/4/ja/][version] で公開
+されている [コントリビュータ協定][homepage] バーlョン 1.4 に適合しています。
```
Comment (ykzts) [inline version/1/4/ja/code_of_conduct.md]: > バーlョン

why? "バージョン" is correct.

**Review by ykzts — CHANGES_REQUESTED** [review]

File: version/1/4/ja/code_of_conduct.md (hunk 2 above)
Comment (TatchNicolas) [inline version/1/4/ja/code_of_conduct.md]: Such a stupid type of me...Thanks for pointing out.

**Review by ykzts — APPROVED** [review]

File: version/1/4/ja/code_of_conduct.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# コントリビュータ協定 行動規範
+
+## 宣誓
+
+オープンで歓迎的な雰囲気の環境を育成するため、私達はコントリビュータお
+よびメンテナとして、年齢、体系、障碍、民族、性自認や性別表現、経験の多
```
Comment (rintaro) [inline version/1/4/ja/code_of_conduct.md]: 体系 => 体型

Comment (TatchNicolas) [inline version/1/4/ja/code_of_conduct.md]: TXT version also had the same typo, already fixed. Thanks.

**Pull request closed without merging**